source/connect/mysql.txt

```diff
@@ -45,25 +66,64 @@ password after the command has been entered.
 
    mysql '--user=reportsUser?source=admin' --default-auth=mongosql_auth -p
 
-This example assumes that the authentication plugin file
-``mongosql_auth.so`` is located in the default MySQL plugin folder.
-The location of the plugin folder varies by platform, but you can
-locate it by running the following command:
+.. note::
+
+   This example assumes that the authentication plugin file
+   ``mongosql_auth.so`` is located in the default MySQL plugin folder.
+   The location of the plugin folder varies by platform, but you can
+   locate it by running the following command:
+
+   .. cssclass:: copyable-code
+   .. code-block:: sh
+
+      mysql_config --plugindir
+
+   ``mysql_config.pl`` can find the plugin directory only on macOS and 
+   Linux hosts.
+
+32-bit Windows
+~~~~~~~~~~~~~~
 
 .. cssclass:: copyable-code
-.. code-block:: sh
+.. code-block:: ps1
 
-   mysql_config --plugindir
+   "C:\Program Files (x86)\MySQL\MySQL Server 5.7\bin\mysql.exe" ^
+     '--user=reportsUser?source=admin' ^
+     --default-auth=mongosql_auth -p
+
+64-bit Windows
+~~~~~~~~~~~~~~
+
+.. cssclass:: copyable-code
+.. code-block:: ps1
+
+   "C:\Program Files\MySQL\MySQL Server 5.7\bin\mysql.exe" ^
+     '--user=reportsUser?source=admin' ^
+     --default-auth=mongosql_auth -p
 
 .. _connect-with-tls-ssl:
 
 Connect from MySQL with Authentication and TLS/SSL
 --------------------------------------------------
 
+.. important::
+
+   The `binary distribution of MySQL Community 
+   <https://dev.mysql.com/doc/refman/5.7/en/openssl-versus-yassl.html>`_ 
+   uses the `yaSSL <https://www.wolfssl.com/products/yassl/>`_ 
+   :abbr:`SSL (Secure Sockets Layer)` library to encrypt connections.
+   MySQL Enterprise uses `OpenSSL <https://www.openssl.org/>`_ which is
+   compatible with |bi|. Use MySQL Enterprise to connect to |bi-short| 
```

Comment:
If taking the above suggestion, can maybe mod this bit to use |bi-short| instead of |bi|.